include/nbl/builtin/glsl/material_compiler/common_invariant_declarations.glsl

```diff
@@ -80,9 +80,19 @@ void nbl_glsl_MC_finalizeMicrofacet(inout nbl_glsl_MC_microfacet_t mf)
 
 #include <nbl/builtin/glsl/format/decode.glsl>
 
-struct nbl_glsl_MC_oriented_material_t
+#define NBL_GLSL_MC_INVALID_EMITTER_ID 0
+
+struct nbl_glsl_MC_emitter_t
 {
 	uvec2 emissive;
+	uvec2 emissionProfile;
+	float normalizeEnergy;
```

Comment:
you don't need this, scaling should already be multiplied into `emissive` also you're messing up the struct alignment with padding